src/extension.ts

```diff
@@ -58,6 +59,9 @@ export async function activate(context: vscode.ExtensionContext) {
     ext.sdkClientBuilder = new DefaultAWSClientBuilder(awsContext)
     ext.toolkitClientBuilder = new DefaultToolkitClientBuilder()
     ext.statusBar = new AWSStatusBar(awsContext, context)
+    // TODO: uncomment when we have a way to surface a telemetry confirmation prompt to users
```

Comment:
Prompt can be added through `vscode.window.showInformationMessage`. This can set a value in the Settings page (create the menu option through `package.json`) and can be set manually through `./shared/settingsConfiguration.ts`'s `writeSetting`. You can then check the value with `readSetting`.

Reply:
Sounds good. Will probably look into adding this through another PR.